crates/cast/tests/e2e/declare.rs

```diff
@@ -85,12 +87,10 @@ async fn wrong_contract_name_passed() {
         .current_dir(CONTRACTS_DIR.to_string() + "/v1/map")
         .args(args);
 
-    let output = String::from_utf8(snapbox.assert().failure().get_output().stderr.clone()).unwrap();
-
-    assert!(
-        output.contains("Failed to find contract nonexistent in starknet_artifacts.json"),
-        "Expected error message not found in stderr: {output}",
-    );
+    snapbox.assert().failure().stderr_matches(indoc! {r#"
+        [..]Failed to find contract nonexistent in starknet_artifacts.json[..]
```

Comment:
should this also include an information about command, to be in line with other outputs?

Reply:
Will change it after the #513 is merged